src/modules/menu-dishes/menu-dishes.module.ts

```diff
@@ -3,9 +3,13 @@ import { TypeOrmModule } from '@nestjs/typeorm';
 import { MenuDishesRepository } from './menu-dishes.repository';
 import { MenuDishesService } from './menu-dishes.service';
 import { MenuDishesController } from './menu-dishes.controller';
+import { MenuCategoryModule } from '../menu-category/menu-category.module';
 
 @Module({
-  imports: [TypeOrmModule.forFeature([MenuDishesRepository])],
+  imports: [
+    MenuCategoryModule,
+    TypeOrmModule.forFeature([MenuDishesRepository]),
```

Comment:
you can add the menu category repository here and you don't need the export in menu category module:
TypeOrmModule.forFeature([MenuDishesRepository, MenuCategorRepository ]),